pep-0585.rst

```diff
@@ -283,38 +283,38 @@ With ``__class_getitem__`` as an identity function::
 The indexing being successful here would likely end up raising an
 exception at a distance, confusing the user.
 
-Disallowing instantiation of parametrized types
+Disallowing instantiation of parameterized types
 -----------------------------------------------
```

Comment:
Whoops, need to add an extra `-` to this line.

Reply:
Saw that, done!